scripts/jenkins/jobs-obs/openstack-trackupstream.yaml

```diff
@@ -155,6 +157,8 @@
               "openstack-horizon-plugin-ironic-ui",
               "openstack-horizon-plugin-magnum-ui",
               "openstack-octavia",
+              "openstack-murano",
+              "python-muranoclient",
```

Comment:
Oh, great point! Thanks for checking this out @bmwiedemann creating the package now...